tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -63,7 +63,7 @@ def create_dag(database_name, table_args, dataset):
             output_table=table,
             chunk_size=chunk_size,
         )
-        aql.truncate(my_table)
+        aql.cleanup([my_table])
```

Comment:
Why have we introduced this change?

Reply:
@tatiana two reasons

1) there are issues in truncate that I'm gonna make tickets for today
2) Truncate empties but doesn't delete the table. We want to delete the table here.

Reply:
BTW: are there more bugs besides  https://github.com/astronomer/astro-sdk/issues/515?